### ISSUE TITLE
Fix width of bars in last two figures showing authorship practices over time

### ISSUE THREAD

**Issue by duffymeg** [duffymeg/DEAuthorshipPoll#8]
Currently, the last two figures (showing number of authors over time and corresponding author position over time) have bars that are different widths, due to missing combinations.

Some possible solutions:
1. http://stackoverflow.com/questions/15367762/include-space-for-missing-factor-level-used-in-fill-aesthetics-in-geom-boxplot
2. http://stackoverflow.com/questions/19778506/consistent-box-width-with-missing-values-in-a-ggplot-box-plot
3. http://stackoverflow.com/questions/16705129/force-boxplots-from-geom-boxplot-to-constant-width

**Comment by duffymeg** [comment]
First attempted fix involved installing github version of ggplots, based on rec via twitter from Hadley Wickham. This has solved the boxwidth issue for the box plot, but now a dodge issue has appeared. On top of that, still having problems with the bar plot. Will have to figure out later.

**Comment by jjvenky** [comment]
So what's the fix you tweeted [here](https://twitter.com/duffy_ma/status/837769139171241984)?
Inquiring minds what to know!

**Comment by duffymeg** [comment]
I'm not sure if I will link correctly, but hopefully this will take you to the fix! I added in zeroes for the missing values (in a kludgy sort of way). I still haven't worked out the issue with the box plot.

https://github.com/duffymeg/DEAuthorshipPoll/commit/e6e7423e92304875277f51dba9f5b5d21d4871a9

**Comment by jjvenky** [comment]
Yes. Thanks. Essentially you're giving the geom something to plot to force the spacing to work. I'll add this to my ever-growing list of things student will some day ask :+1:
